test/src/test/java/org/corfudb/integration/CorfuReplicationReconfigurationIT.java

```diff
@@ -227,6 +228,147 @@ public void testActiveClusterReset() throws Exception {
         assertThat(remainingEntriesToSend).isEqualTo(0L);
     }
 
+    /**
+     * This test verifies that concurrent Snapshot Syncs are handled gracefully on the Standby.  If the apply phase
+     * of a snapshot sync is ongoing, a new snapshot sync is rejected, avoiding any race.
+     * @throws Exception
+     */
+    @Test
+    public void testConcurrentSnapshotSyncOnStandby() throws Exception {
```

Comment:
discussed offline but just to note, we should look Into finding deterministic way to test sync drops during apply phase